toolsrc/src/vcpkg/install.cpp

```diff
@@ -617,10 +619,42 @@ namespace vcpkg::Install
                             config_files[find_package_name] = root;
                     }
                 }
+                if (Strings::case_insensitive_ascii_contains(suffix, "/lib/") ||
+                    Strings::case_insensitive_ascii_contains(suffix, "/bin/"))
+                {
+                    if (!Strings::ends_with(suffix, ".pc")) is_header_only = false;
```

Comment:
This is broken, since if your file tree is:

```
/share/
  x.pc
```

then your file list will be

```
<triplet>/share/
<triplet>/share/x.pc
```

you probably want

```suggestion
                    if (!Strings::ends_with(suffix, ".pc") && !Strings::ends_with(suffix, "/")) is_header_only = false;
```